v3/src/components/graph/components/casedots.tsx

```diff
@@ -28,18 +29,28 @@ export const CaseDots = function CaseDots(props: {
     currPos = useRef({x: 0, y: 0}),
     target = useRef<any>()
 
-  const onDragStart = useCallback((event: MouseEvent) => {
-      enableAnimation.current = false // We don't want to animate points until end of drag
-      target.current = select(event.target as SVGSVGElement)
-      const aCaseData: CaseData = target.current.node().__data__
-      if (aCaseData && target.current.node()?.nodeName === 'circle') {
-        target.current.transition()
-          .attr('r', dragPointRadius)
-        setDragID(aCaseData.caseID)
-        currPos.current = {x: event.clientX, y: event.clientY}
-        handleClickOnDot(event, aCaseData.caseID, dataset)
-      }
-    }, [dragPointRadius, dataset, enableAnimation]),
+  const randomlyDistributePoints = useCallback((cases?: CaseData[] | undefined) => {
+    const uniform = randomUniform()
```

Comment:
This is probably fine, and I see it was carried over from the previous state of the code, but why don't we just use `randomUniform()` on line 37 instead of creating the new `uniform()` here and using that instead?